js/modules/k6/metrics/metrics.go

```diff
@@ -65,7 +65,18 @@ func newMetric(ctxPtr *context.Context, name string, t stats.MetricType, isTime
 	}
 
 	rt := common.GetRuntime(*ctxPtr)
-	return common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr), nil
+	binded := common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr)
+	o := rt.NewObject()
+	err := o.DefineAccessorProperty("name", rt.ToValue(func() interface{} {
```

Comment:
This could be `DefineDataProperty` instead, no? Since we don't need to call a function for it.

Reply:
resolved in 20c22d9